### ISSUE TITLE
Links on Moodle

### ISSUE THREAD

**Issue by erinflint** [Montana-Media-Arts/120_CreativeCoding#170]
For some reason my links on moodle have a 50/50 chance of working. My readme links almost always work but the link I make for my sketches work about half of the time. I upload the same way each time and am unsure as to what is happening. Any help is appreciated!

**Comment by DVSnell** [comment]
Looking at your moodle submission, what you're trying to link to is a local file on your system.  I peeked at your Repo and I think I see the issue.

Each assignment needs to have the `index.html` file associated with it inside the HW folder.  For example, your 'HW-8' folder has your sketch and `p5_lib` folders, but is lacking an `index.html` and `readme.md`.

Your sketch file needs the html file to be displayed in a web browser since the `index.html` file is what your browser actually displays.  If you move the `index.html` file you submitted into the `HW-8` folder then commit and upload, it should fix the problem with some adjusted links.  The link you'll want to use after pushing to actually view your Live Site Sketch for HW-8 will be 'https://erinflint.github.io/120-Work/HW-8' with `https://github.com/erinflint/120-Work/tree/master/HW-8` being your Repo link when it contains the `readme.md` for HW-8.

Basically, every assignment should look like this in Atom:
![directorystructure](https://user-images.githubusercontent.com/39629030/47130207-ca20cb00-d255-11e8-9da1-01417ed92a71.png)

And like this on the Github site:
![github directory](https://user-images.githubusercontent.com/39629030/47130278-271c8100-d256-11e8-86bb-08672b98cfd3.png)

Note the `images` folder is just used for `readme.md` screenshots and you might not have any.

**Comment by erinflint** [comment]
Thank you thank you!!!
